be/src/olap/rowset/segment_v2/plain_page.h

```diff
@@ -157,6 +157,44 @@ class PlainPageDecoder : public PageDecoder {
         return Status::OK();
     }
 
+    Status seek_at_or_after_value(const void* value, bool* exact_match) override {
```

Comment:
Could we unify the `seek_at_or_after_value` in `plain_page.h` and `bitshuffle_page.h`